packages/jni-swig-stub/src/main/jni/realm_api_helpers.cpp

```diff
@@ -337,6 +338,76 @@ void app_complete_result_callback(void* userdata, void* result, const realm_app_
     }
 }
 
+jobject create_api_key_wrapper(JNIEnv* env, const realm_app_user_apikey_t* key_data) {
+        auto id_size = sizeof(key_data->id.bytes);
+        static JavaClass api_key_wrapper_class(env, "io/realm/kotlin/internal/interop/sync/ApiKeyWrapper");
+        static JavaMethod api_key_wrapper_constructor(env, api_key_wrapper_class, "<init>", "([BLjava/lang/String;Ljava/lang/String;Z)V");
+        jbyteArray id = env->NewByteArray(id_size);
+        env->SetByteArrayRegion(id, 0, id_size, reinterpret_cast<const jbyte*>(key_data->id.bytes));
+        jstring key = to_jstring(env, key_data->key);
+        jstring name = to_jstring(env, key_data->name);
+        jboolean disabled = key_data->disabled;
+        return env->NewObject(api_key_wrapper_class,
+                                                     api_key_wrapper_constructor,
```

Comment:
Something about indentation is not right here. They should be at the same level as `api_key_wrapper_class`